frontend/src/components/General/Navbar.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = (props: {
+    brand: { name: string; to: string },
+    links: Array<{ name: string, to: string }>
+}) => {
+    const { brand, links } = props;
+    const NavLinks = () => links.map((link: { name: string, to: string }) => 
+            <NavLink
+                to={link.to}
+                key={link.name}
+                className={({ isActive }) => "mr-5 " + (isActive ? "font-bold" : "font-semibold")}
+            >
+                {link.name}
+            </NavLink>
```

Comment:
Instead of using li, it is better to use NavLink for navigation bars, it has some useful implementations. For that it is necessary to have the entire react-router-dom implementation, that's why I implemented it in your pr.